smart-contracts/contracts/DigitalaxAuction.sol

```diff
@@ -198,14 +238,15 @@ contract DigitalaxAuction is Context, ReentrancyGuard {
      */
     function resultAuction(uint256 _garmentTokenId) external nonReentrant {
 
-        // TODO check this should be admin and smart contract role check
         // Admin only resulting function
-        require(accessControls.hasAdminRole(_msgSender()), "DigitalaxAuction.resultAuction: Sender must be admin");
+        require(
+            accessControls.hasAdminRole(_msgSender()) || accessControls.hasSmartContractRole(_msgSender()),
+            "DigitalaxAuction.resultAuction: Sender must be admin"
+        );
 
         // Check the auction to see if it can be resulted
         Auction storage auction = auctions[_garmentTokenId];
 
-        // FIXME AMG - drop this require as the next one will do the job?
         // Check the auction real
```

Comment:
thoughts on the FIXME?

Reply:
ah - removed - sorry

Reply:
I left in to give better validation errors when resulting thats all